src/main/java/storage/Storage.java

```diff
@@ -9,17 +11,21 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.time.LocalDate;
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import static data.TaskManager.addTask;
+import static data.TaskManager.parseTaskType;
+import static data.TaskType.DEADLINE;
+import static data.TaskType.EVENT;
+
 public class Storage {
 
     public static final Path FILE_PATH = Path.of("./save/tasks.txt");
-    private static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+    private static final Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
```

Comment:
We should use capitalized names for variables marked as "final" as per the convention